docs/content/en/highlights/2021-07-13-helm-customConfig.md

```diff
@@ -0,0 +1,324 @@
+---
+date: "2021-07-13"
+title: "Introduction of `customConfig` and deprecation notice for TOML based config keys"
```

Comment:
```suggestion
title: "Introduction of `customConfig` Helm option and deprecation notice for TOML based config keys"
```

Feel free to adjust. Just trying to make the title more obvious that it's a Helm change.